docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[indicators-of-compromise]]
+= Indicators of compromise
+
+The Intelligence page collects data from enabled threat intelligence feeds and provides you with a centralized view of indicators. This topic offers definitions of threat intelligence and indicators, helps you get started using the Intelligence page, and explains how to work with indicators.
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+If you are unfamiliar with threat intelligence, think of it as a research function that helps an organization make informed decisions when taking actions to protect itself. An organization's security operation center (SOC) team is usually responsible for managing threat intelligence. A SOC team's goals are to understand current and emerging threats so they can recommend actions that will prevent, stop, and mitigate threats to an organization.
+
+An indicator, also referred to as an Indicator of Compromise (IoC), is a piece of information that represents a known malicious threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
```

Comment:
```suggestion
An indicator, also referred to as an Indicator of Compromise (IoC), is a piece of information associated with a known threat or vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
```

Reply:
A few tweaks to @benironside's edit. I kinda think "reported" is good to include since a lot of threat intel seems to depend on reporting of info:
```suggestion
An indicator, also referred to as an indicator of compromise (IoC), is a piece of information associated with a known threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
```

Reply:
I've seen indicators of compromise written both ways but am totally fine using the all lowercase form and treating it like a common noun